docker/worker/run.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+#set -e
+
+if [ -z $1 ]; then
+  echo "Error: define the number of workers"
+  echo "Usage: $0 <number-of-workers>"
+  exit
+fi
+
+for i in $(seq $1); do
```

Comment:
I know, `$1` is just supposed to be a number. However, it would still make sense to quote it.

Reply:
shellcheck would complain about that but so far we only look into shellscripts in the directory `script/`, see https://github.com/os-autoinst/openQA/blob/master/Makefile#L252

I recommend that we either do not care too much about the style for now or we include these shell scripts in shellcheck as well.

Reply:
I don't think automatic checks are a requirement to have quotes here.

Reply:
of course not. I just wanted to help ilausuch get his PR in sooner but still prevent that we are getting frustrated with inconsistent shell code style.